lib/modules/manager/gomod/update.ts

```diff
@@ -46,9 +46,15 @@ export function updateDependency({
       updateLineExp = regEx(/(?<depPart>go)(?<divider>\s+)[^\s]+/);
     }
     if (depType === 'replace') {
-      updateLineExp = regEx(
-        /^(?<depPart>replace\s+[^\s]+[\s]+[=][>]+\s+)(?<divider>[^\s]+\s+)[^\s]+/
-      );
+      if (upgrade.managerData.multiLine) {
+        updateLineExp = regEx(
+          /^(?<depPart>\s+[^\s]+[\s]+[=][>]+\s+)(?<divider>[^\s]+\s+)[^\s]+/
+        );
+      } else {
+        updateLineExp = regEx(
+          /^(?<depPart>replace\s+[^\s]+[\s]+[=][>]+\s+)(?<divider>[^\s]+\s+)[^\s]+/
```

Comment:
```suggestion
          /^(?<depPart>replace\s+[^\s]+[\s]+=>+\s+)(?<divider>[^\s]+\s+)[^\s]+/
```
isn't this the same regex?
- are you sure `>` can occur multiple times?

Reply:
No just once. This regex is based on the original